README.md

```diff
@@ -1,8 +1,6 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-
-> Description the project.
+Hello-Microverse
 
```

Comment:
- [ ] Great job using the README.md Template 💯 However, I suggest adding a description to your project 👌